cvat/apps/engine/models.py

```diff
@@ -861,6 +861,10 @@ def get_specific_attributes(self):
     def get_key_file_path(self):
         return os.path.join(self.get_storage_dirname(), 'key.json')
 
+    @property
+    def has_at_least_one_manifest(self) -> bool:
```

Comment:
`manifest_count(self)` looks more logical

Reply:
> We don't need the information about the number of attached manifests anywhere. We need to know if is there at least one.

If you return count value, the semantics of the function will not change, you can use it as before `if xxx.manifest_count(): ...` or just compare it with an int value.

Reply:
If return count value we get a bit more universal method. For example, in the future I will need information about the total number of manifests. But I agree this is a insignificant comment, and the method doesnt affect the functionality of pr